.eslintrc.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
     "no-underscore-dangle": 0,
     "simple-import-sort/sort": "warn",
     "sort-imports": "off",
+    "no-unused-vars": "off",
```

Comment:
i don't think we should do this, it will miss major linting, when we write code

Reply:
no, in typescript-eslint it's recommended to do it - [https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md#how-to-use](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md#how-to-use)

because we're using "@typescript-eslint/no-unused-vars" rules here. Checked in the js-examples as well, works as expected.